src/index.ts

```diff
@@ -130,9 +130,20 @@ export function useApiLogger(options?: IUseApiLoggerOptions | null | undefined):
 
         let data = args[0];
         if (typeof data === 'string') {
+            let metadata: any;
+            if (args.length > 0) {
```

Comment:
@zulfiqarjunejo 

I think here is still a small bug.

In my opinion it should be `args.length > 1`, because if `data` is a string, which is the first argument of `args`, `args` will always have a length of at least 1

see also: https://github.com/egomobile/api-log/pull/8#issuecomment-1024032232

Reply:
Yes, I also thought about it. But it's not a problem because if data is a string, it will just create an empty metadata object. And we are anyway skipping the entry [0], so we are good.